cli50/__main__.py

```diff
@@ -252,25 +252,29 @@ def ports(container):
     ]).decode("utf-8").rstrip()
 
 
-def pull(image, tag):
+def pull(image):
     """Pull image as needed."""
     try:
 
         # Get digest of local image, if any
-        digest = subprocess.check_output(["docker", "inspect", "--format", "{{index .RepoDigests 0}}", f"{image}:{tag}"],
+        digest = subprocess.check_output(["docker", "inspect", "--format", "{{index .RepoDigests 0}}", image],
                                          stderr=subprocess.DEVNULL).decode("utf-8").rstrip()
 
         # Get digest of latest image
         # https://stackoverflow.com/a/50945459/5156190
-        response = requests.get(f"https://hub.docker.com/v2/repositories/{image}/tags/{tag}").json()["images"][0]
+        repository, tag = image.split(":") if ":" in image else image, "latest"
+        if "/" not in repository:
+            repository = "library/" + repository
+        print(f"https://hub.docker.com/v2/repositories/{repository}/tags/{tag}")
```

Comment:
Should this be removed?

Reply:
Hm, personally, not sure I love `repo = repo`.

Reply:
Ha, oops, removing.